tools/dev/loki-boltdb-storage-s3/dev.dockerfile

```
@@ -1,6 +1,6 @@
 FROM golang:1.20.4
```

Should we also update the GO version?

```suggestion
FROM golang:1.21.3
```